doc/guides/triaging-for-lts-releases.md

```diff
@@ -0,0 +1,127 @@
+# Triaging for LTS Releases
+
+### Set up your repo
+
+- You will need collaborator access.
+
+- Make sure nobody else is currently backporting to the same LTS branch.
+
+- Clone node
+
+  ```bash
+  git clone git@github.com/nodejs/node --origin upstream
+  ```
+
+- Make sure you're up to date with staging branch (where N is an LTS release e.g. 6,8,10)
+  ```bash
+  git fetch --all
+  git checkout -B vN.x-staging upstream/vN.x-staging
+  ```
+
+- Run the tests on the  to make sure everything works:
+  ```bash
+  ./configure && make -j4 test
+  ```
+
+- Install branch-diff (and set up the key)
+  ```bash
+  npm install -g branch-diff
+  ```
+
+### Triage PRs
+
+Go through PRs with the label `vN.x`. e.g. [PRs with the v8.x label](https://github.com/nodejs/node/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3Av8.x)
+
+For each PR:
+
+- Run or check that there is a passing CI
+- Check approvals (you can approve yourself)
+- Check SemVer
+- Check the PR matches the wasn't changed from `master` commit
+- If there are merge conflicts, comment in the issue asking the PR author to rebase.
+- Land if okay
+  - Add git alias if you haven't already:
+  ```bash
+  git config --global alias.pa '!curl -L $1.patch | git am -3 -S --whitespace=fix #'
+  ```
+  - Patch Pull Request onto staging branch:
+  ```bash
+  git pa https://github.com/nodejs/node/pull/12345
+  ```
+  - If it doesn't apply cleanly then abort with:
+  ```bash
+  git am --abort
+  ```
+  - If it lands and tests pass, then add the `Backport-PR-URL:` line to each commit message:
+  ```bash
+  # If a single commit:
+  git commit --amend
+
+  # If multiple commits:
+  git rebase -i @~4 # If 3 commits.
+  # Then change pick to reword for each commit
+  #
+
+  # Add this line to each commit under the PR-URL:
+  # Backport-PR-URL: https://github.com/nodejs/node/pull/12345
+  ```
+  - Test the commit and push it
+    ```bash
+    ./configure && make -j4 test
+    git push
+    ```
+  - Close the backport PR with `Landed in $(git rev-parse --short HEAD)`, update the label on the original PR from `backport-requested-vX.x` to `backported-to-vX.x`.
+
+### Triage commits
```

Comment:
I'd say this should come at the top before the `Triage PRs` section potentially?